src/main/resources/main.css

```diff
@@ -1,7 +1,7 @@
 /*#region Package information*/
 #packageInfoHeader{
     position: fixed;
-    top: 5%;
+    top: 3%;
```

Comment:
Moved the header a little up so it isn't too close to the textbox. 
![example1](https://user-images.githubusercontent.com/71157283/98431618-0c0f4a80-2085-11eb-917b-7ecbc3ed85a5.PNG)
![example2](https://user-images.githubusercontent.com/71157283/98431621-0e71a480-2085-11eb-8055-f33a1ecb94de.PNG)